packages/toolpad-core/src/Crud/List.tsx

```diff
@@ -1,7 +1,7 @@
 'use client';
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { styled } from '@mui/material';
+import { NoSsr, styled } from '@mui/material';
```

Comment:
We have a better version of [NoSsr](https://github.com/mui/toolpad/blob/077f338f38acef9557a40b648a48cc26af0a92ba/packages/toolpad-studio/src/components/NoSsr.tsx#L13) in toolpad studio.

Reply:
cool, added in https://github.com/mui/toolpad/pull/4907/commits/a2e04860801b02e23ab4cc2a569baf29b9bfe835